server/fleet/datastore.go

```diff
@@ -871,6 +871,13 @@ type Datastore interface {
 	GetMDMAppleBootstrapPackageMeta(ctx context.Context, teamID uint) (*MDMAppleBootstrapPackage, error)
 	// GetMDMAppleBootstrapPackageBytes returns the bytes of a bootstrap package with the given token
 	GetMDMAppleBootstrapPackageBytes(ctx context.Context, token string) (*MDMAppleBootstrapPackage, error)
+	// GetMDMAppleBootstrapPackageSummary returns an aggregated summary of
+	// the status of the bootstrap package for hosts in a team.
+	GetMDMAppleBootstrapPackageSummary(ctx context.Context, teamID uint) (MDMAppleHostStatusSummary, error)
```

Comment:
Nit, but just wondering if we should/could standardize on an approach. Seeing the diff like that makes it apparent that `GetMDMAppleHostsProfilesSummary` and `GetMDMAppleBootstrapPackageSummary` have the same signature save for one receiving `teamID *uint` instead of `teamID uint`, and returning `*MDMAppleHostStatusSummary` instead of `MDMAppleHostStatusSummary`.